test/spec/marfeelTools_spec.js

```diff
@@ -12,7 +12,8 @@
  * from Marfeel Solutions SL.
  */
 
-import { filterBidsBySizes } from './marfeelTools';
+import { isBidSizeAllowed, add1x1IfAllowed } from './marfeelTools';
```

Comment:
You're importing the private helper instead of `getAllowedSizes`. Did this test pass?

```suggestion
import { isBidSizeAllowed, getAllowedSizes } from './marfeelTools';
```